lib/cisco_node_utils/bridge_domain.rb

```diff
@@ -121,6 +121,30 @@ def destroy
     #                      PROPERTIES                      #
     ########################################################
 
+    # Bridge-Domain name assigning case
+    # bridge-domain 100
+    #   name PepsiCo
+    def bd_name
+      res = config_get('bridge_domain', 'bd_name', bd: @bd_ids)
+      return default_bd_name unless res
+      res.compact.join
```

Comment:
See my yaml comment related to this: https://github.com/cisco/cisco-network-node-utils/pull/302/files#diff-dea5252853dd670210b2588b4305051eR49

...with that change you should be able to just call config_get() by itself:

```
  def bd_name
    config_get('bridge_domain', 'bd_name', bd: @bd_ids)
  end
```